### PR TITLE
setzer: update to 0.4.1.

### PR DESCRIPTION
**Pull request by Bleznudd**
_For the setzer update commit:_
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [X] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

---

_For the pdfminersix commit:_
#### General
- [X] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [X] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/python3-pdfminersix/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+short_desc="Tool for extracting information from PDF documents"
+maintainer="Bleznudd <bleznudd@pm.me>"
+license="MIT"
+homepage="https://github.com/pdfminer/pdfminer.six"
+distfiles="${homepage}/archive/refs/tags/${version}.tar.gz"
+checksum=0e9c8b0a9c40c2aee044c3091ad6b18b5e6279722e64b6b975f8aedc26bf8e38
+
+if [ "${CROSS_BUILD}" ]; then
+	hostmakedepends="python3"
+fi
```
Comment (ericonr) [inline srcpkgs/python3-pdfminersix/template:18]:
```suggestion
```

File: srcpkgs/python3-pdfminersix/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'python3-pdfminersix'
+pkgname=python3-pdfminersix
```
Comment (ericonr) [inline srcpkgs/python3-pdfminersix/template:2]: Why not use `python3-pdfminer.six`?

File: srcpkgs/python3-pdfminersix/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+version=20201018
+revision=1
+wrksrc=pdfminer.six-${version}
+build_style=python3-module
+makedepends="python3-setuptools"
+depends="python3 python3-cryptography"
```
[the remark this answers is not in the document]
Comment (ericonr) [reply srcpkgs/python3-pdfminersix/template:8]: I think `python3-chardet` should also be in `depends`.

Then add `checkdepends="$depends"` and see how it goes. Might need to add `tox` and `python3-nose` as well.

[the remark this answers is not in the document]
Comment (Bleznudd) [reply srcpkgs/python3-pdfminersix/template]: You're correct on `python3-chardet`.

About the checks, should I add `tox` and `python3-nose` only if the build server fails without them?

**Comment by ericonr** [comment]
Could you add this patch? https://github.com/pdfminer/pdfminer.six/commit/61300eef70bf2b32d300005f011bdf9b48660740.patch

File: srcpkgs/python3-pdfminersix/template (hunk 3 above)
[the remark this answers is not in the document]
Comment (ericonr) [reply srcpkgs/python3-pdfminersix/template]: I think it'd be best to add them unconditionally.

**Comment by Bleznudd** [comment]
> Could you add this patch? https://github.com/pdfminer/pdfminer.six/commit/61300eef70bf2b32d300005f011bdf9b48660740.patch

Since it's my first time writing a patch file, is everything correct?

**Comment by ericonr** [comment]
Perfect, thanks :)

Will merge locally because GH UI is bugged :/

**Pull request closed without merging**